src/frontend/web_application/src/components/form/InputFile/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { FieldErrors } from '../';
+import Button from '../../Button';
+import Icon from '../../Icon';
+
+import './style.scss';
+
+const InputFile = ({ onChange, className, accept, errors, __ }) => (
+  <div className={classnames('m-input-file', className)}>
+    <label htmlFor="files" className="m-input-file__label">
+      <Button className="m-input-file__label__button" icon="plus" shape="plain" />
+      <span className="m-input-file__label__text">{__('input-file.add_a_file.label')}</span>
+      <span className="m-input-file__label__icon"><Icon type="folder" /></span>
+      <input
+        id="files"
```

Comment:
it will be a problem w/ multiple instance of InputFile in a form. What about uuid ?